l10n_it_fatturapa_out/migrations/16.0.1.6.0/pre-migrate.py

```diff
@@ -0,0 +1,12 @@
+#  Copyright 2025 Michele Di Croce - Stesi Consulting
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    # Clean old transient data
+    drop_old_wizard_export_sql = "DELETE FROM wizard_export_fatturapa;"
```

Comment:
Temo di non aver mai visto eliminare un transient nel migrate, in ogni caso male non penso faccia, il `;` finale però non mi risulta servire.